### PR TITLE
Add Query support for PostgreSQL

### PR DESCRIPTION
**Pull request by r-c-correa**
## Description
Add Query support for PosgreSQL

## Issue reference
#2159

### CODE REVIEW

**Comment by yaron2** [comment]
@r-c-correa please change the target branch to be `v1.7`. thanks!

**Comment by msfussell** [comment]
@r-c-correa please change the target branch to be v1.7

File: daprdocs/content/en/concepts/overview.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Today we are experiencing a wave of cloud adoption. Developers are comfortable w
 
 This is where Dapr comes in. Dapr codifies the *best practices* for building microservice applications into open, independent APIs called building blocks, that enable you to build portable applications with the language and framework of your choice. Each building block is completely independent and you can use one, some, or all of them in your application.
 
-Using Dapr you can incrementally migrate your existing applications to a microserivces architecture, thereby adopting cloud native patterns such scale out/in, resilency and independent deployments.
+Using Dapr you can incrementally migrate your existing applications to a microserivces architecture, thereby adopting cloud native patterns such scale out/in, resiliency and independent deployments.
```
Comment (msfussell) [inline daprdocs/content/en/concepts/overview.md]:
```suggestion
Using Dapr you can incrementally migrate your existing applications to a microservices architecture, thereby adopting cloud native patterns such scale out/in, resiliency and independent deployments.
```

**Comment by msfussell** [comment]
@r-c-correa - Looks like you pulled in many other files as part of this PR. Can you remove all the other changes other than your for the Query update to the table? Thanks

**Comment by r-c-correa** [comment]
Changing target branch to be v1.7 messed up all the commits, I'm going to close this pull request and open a new. Sorry about that 😢

**Pull request closed without merging**